dashboards/rabbitmq/rabbitmq-prometheus.json

```diff
@@ -0,0 +1,889 @@
+{
+  "category": "CUSTOM",
+  "displayName": "RabbitMQ Overview",
+  "etag": "fb01dfe9affb8c67bb9f22f6e3fd98ed",
```

Comment:
Please remove this field, this is specific to the instance of the dashboard the template was copied from.